app/src/common/shared/org/mozilla/vrbrowser/ui/widgets/NavigationBarWidget.java

```diff
@@ -471,10 +472,20 @@ private void enterResizeMode() {
         closeFloatingMenus();
     }
 
-    private void exitResizeMode(boolean aCommitChanges) {
+
+    enum ResizeAction {
+        KEEP_SIZE,
+        RESTORE_SIZE
```

Comment:
Are we using `enum` now? Do they no longer bloat the APK? We are now using both `enum` and `final int` in different places. `final int` I guess makes sense for JNI calls. Not sure if there is a benefit to `enum`?